x/ethbridge/keeper/msg_server.go

```diff
@@ -2,10 +2,15 @@ package keeper
 
 import (
 	"context"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
```

Comment:
please goimport this file. grouping should be

std libs

thid party libs

sifnode imports